profile/profile.go

```diff
@@ -715,6 +715,36 @@ func (s *Sample) HasLabel(key, value string) bool {
 	return false
 }
 
+// SetNumLabel sets the specified key to the specified value for all samples in the
+// profile.
+func (p *Profile) SetNumLabel(key string, value []int64) {
```

Comment:
I think this helper should take the unit parameter as in vast majority of cases numeric labels need to specify the unit. Besides the general advantage of units giving the numeric labels physical meaning, handling of zero values for numeric labels subtly depends on the presence of a unit - https://github.com/google/pprof/pull/619.

Reply:
Ah yeah, that makes sense! Fixed.